pom.xml

```diff
@@ -106,6 +106,21 @@
       <version>4.0.3.Final</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>build.buf.gen</groupId>
+      <artifactId>lennysgarage_test-kessel_grpc_java</artifactId>
```

Comment:
This is fine for PoC purposes, but we just need to make sure that we can have a suitably generic kessel artifact id.

Reply:
Yup, right now it was left in for testing purposes, will be prefixed with `project-kessel_relations-api` when https://github.com/project-kessel/relations-api/pull/118 is merged and protos are pushed.